microprofile/tests/tck/tck-cdi-lang-model/src/test/java/io/helidon/microprofile/cdi/lang/model/tck/LangModelExtension.java

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
CDI Lang Model comes with one example of how to execute the test. This class and LangModelTckTest are basically a copy-paste of the example.